client/task-list/tasks/payments/RemotePayments/index.js

```diff
@@ -73,6 +67,26 @@ export const RemotePayments = ( { query } ) => {
 		} );
 	};
 
+	const getInitiallyEnabledMethods = () =>
+		methods.reduce( ( acc, method ) => {
+			acc[ method.key ] = method.isEnabled;
+			return acc;
+		}, {} );
+
+	const [ enabledMethods, setEnabledMethods ] = useState(
```

Comment:
Maybe we can add a todo comment here. I think ideally we can rely on the new payments data store in #6918 to track enabled payment methods and optimistically update that store when enabling methods.